feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/base_vrf_selection_test.go

```diff
@@ -155,72 +163,71 @@ func configInterfaceDUT(i *oc.Interface, me, peer *attrs.Attributes, subintfinde
 }
 
 // Configure Network instance on the DUT
-func configNetworkInstance(name string, peer *attrs.Attributes) *oc.NetworkInstance {
+func configNetworkInstance(name string, intf string, id string) *oc.NetworkInstance {
 	d := &oc.Root{}
 	ni := d.GetOrCreateNetworkInstance(name)
 
 	ni.Type = oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L3VRF
-	static := ni.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, *deviations.StaticProtocolName)
-	ipv4Nh := static.GetOrCreateStatic("0.0.0.0/0").GetOrCreateNextHop("0")
-	ipv4Nh.NextHop, _ = ipv4Nh.To_NetworkInstance_Protocol_Static_NextHop_NextHop_Union(peer.IPv4)
-	ipv6Nh := static.GetOrCreateStatic("::/0").GetOrCreateNextHop("0")
-	ipv6Nh.NextHop, _ = ipv6Nh.To_NetworkInstance_Protocol_Static_NextHop_NextHop_Union(peer.IPv6)
-	ipv4Nh.Recurse = ygot.Bool(true)
-	ipv6Nh.Recurse = ygot.Bool(true)
+	niIntf := ni.GetOrCreateInterface(intf)
+	niIntf.Id = ygot.String(id)
+	niIntf.Interface = ygot.String(intf)
 
 	return ni
 }
 
+// configdefaultRoute configures a static route in DEFAULT network-instance.
+func configdefaultRoute(t *testing.T, dut *ondatra.DUTDevice, v4prefix string, v4nexthop string, v6prefix string, v6nexthop string) {
```

Comment:
camelcase fixes: configDefaultRoute, v4Prefix, v4NextHop, v6Prefix, v6NextHop

Reply:
also, no need to repeat the "string" type over all those parameters. It can just be:

```
func configDefaultRoute(t *testing.T, dut *ondatra.DUTDevice, v4Prefix , v4NextHop, v6Prefix, v6NextHop string) {
```

Reply:
done in new commit.